lib/main.ts

```diff
@@ -72,6 +73,13 @@ declare namespace kinde {
     export function denyAccess(reason: string): void;
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace mfa {
+    export function setEnforcementPolicy(
+      mfaEnforcementPolicy: MFAEnformcementPolicy,
```

Comment:
@DanielRivers slight typo of `MFAEnformcementPolicy` should be`MFAEnforcementPolicy`